cac_message/src/Form/CacMessageForm.php

```diff
@@ -0,0 +1,121 @@
+<?php  
+/**  
+ * @file  
+ * Contains Drupal\cac_message\Form\CacxMessageForm.  
+ */  
+namespace Drupal\cac_message\Form;  
+use Drupal\Core\Form\ConfigFormBase;  
+use Drupal\Core\Form\FormStateInterface; 
+use Drupal\Core\Routing\RouteProviderInterface;
+
+class CacMessageForm extends ConfigFormBase {  
+/**  
+   * {@inheritdoc}  
+   */  
+  protected function getEditableConfigNames() {  
+    return [  
+      'cac_message.adminsettings',  
+    ];  
+  }  
+
+  /**  
+   * {@inheritdoc}  
+   */  
+  public function getFormId() {  
+    return 'cac_message_form';  
+  }
+
+
+  /**  
+   * {@inheritdoc}  
+   */  
+	  public function buildForm(array $form, FormStateInterface $form_state) {  
+	    $config = $this->config('cac_message.adminsettings');  
+		$form['welcome_message'] = [
+			'#type' => 'text_format',
+			'#title' => $this->t('Welcome message'),
+			'#description' => $this->t('Display message to users'),
+			'#default_value' => $config->get('welcome_message')['value'],
+			'#required' => TRUE,
+			'#attributes' => array(
+				'class' => array('field_info'),
+				'rows' => '5',
+				'cols' => '200',		  	
+			),
+			'#wysiwyg' => TRUE,	   	
+		];
+	   
+	    $form['message_path'] = [  
+			'#type' => 'textarea',  
+			'#title' => $this->t('Welcome message display path'),  
+			'#description' => $this->t('Welcome message display to specific path. Please add url start with slash and one url per line.<br/>No wildcard (*) supported.'),  
+			'#default_value' => $config->get('message_path'), 
+			'#required' => TRUE,
+			'#maxlength' => 150,
```

Comment:
We should not be limiting a number of characters... we might need to input a lot of paths

Reply:
@tarzan0820 , 
Removed maxlenth